apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/EphemeralTransaction.cs

```diff
@@ -45,28 +42,46 @@ public Task<long> ExecuteDmlAsync(ExecuteSqlRequest request, CancellationToken c
 
             async Task<long> Impl()
             {
-                using (var transaction = await _connection.BeginTransactionImplAsync(_transactionOptions, TransactionMode.ReadWrite, cancellationToken).ConfigureAwait(false))
+                using (var transaction = await _connection.BeginTransactionImplAsync(
+                    SpannerConnection.s_readWriteTransactionOptions, TransactionMode.ReadWrite, cancellationToken).ConfigureAwait(false))
```

Comment:
If s_readWriteTransctionOptions is exposed internally, we should give it a better name... and potentially move it out of SpannerConnection. That can happen later, of course.